.github/workflows/helm-release.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    tags:
```

Comment:
Should we build helm charts only at specific tag?

like:
`    tags:
      - 'v*.*.*'
`
@c121914yu

Reply:
After discussion with @c121914yu , we should change matched tags here @LinuxSuRen

Reply:
No problem from me. So you perfer to use `'v*.*.*`?

Reply:
To match: https://github.com/labring/FastGPT/blob/main/.github/workflows/fastgpt-image.yml#L8 ?

Reply:
Do you recommend to add the file path filter or not? https://github.com/labring/FastGPT/blob/911512b36d7353ebe6cc16428976210dcd8477d3/.github/workflows/fastgpt-image.yml#L5

Such as:

```yaml
on:
  workflow_dispatch:
  push:
    paths:
      - 'files/helm/fastgpt/**'
    tags:
      - 'v*.*.*'
```

Reply:
Seems just cannot match these paths, so keep `*` just works.